src/ethereum/index.ts

```diff
@@ -44,6 +44,8 @@ export default class EthereumWallet implements IWallet {
   coin: EthCoinData;
   evmAddress: string;
   accountIndex: number;
+  //L1 fees may fluctuate upto 25% while broadcasting https://help.optimism.io/hc/en-us/articles/4416677738907-What-happens-if-the-L1-gas-price-spikes-while-a-transaction-is-in-process
+  optimismL1MaxVariation = 1.25;
```

Comment:
If the variable represents variation then the value should be `0.25` representing `25%`.

Reply:
It is meant to represent max value the l1 fee can scale upto